mwa_trigger/parse_xml.py

```diff
@@ -75,21 +68,28 @@ def get_source_types(telescope, event_type, source_name, v):
             # Check if the name is within the "name" string since MAXI does stupid things sometimes
             if f in str(source_name).lower():
                 flare_star =True
+    if flare_star:
+        return "FS"
 
-    #Check for Gravitational Waves
-    if telescope == "LVC":
-        gw = True
-    else:
-        gw = False
-
-    # Check for neutrinos
-    neutrino = False
-    if telescope == "Antares":
-        neutrino = True
-    elif telescope == "AMON" and event_type == "ICECUBE_GOLD":
-        neutrino = True
+    # Check for GRB
+    grb = False
+    if telescope == "SWIFT":
+        # check to see if a GRB was identified
+        try:
+            grb = v.find(".//Param[@name='GRB_Identified']").attrib['value']
+        except AttributeError:
+            logger.error("Param[@name='GRB_Identified'] not found in XML packet - discarding.")
+            grb = False
+        grb = bool(grb) # Make sure it's not a string
+    elif telescope == "Fermi":
+        #grb = False   # Ignore all Fermi triggers
+        grb = True
+        # I could put the most likely index here but it's easier to log in the trigger logic
```

Comment:
If FERMI thinks it's *not* a GRB why do we not capture that here?
eg. If FERMI says that it's 100% a solar flare, why not capture that here?

Reply:
I put this below the flare star logic so if it is a Fermi flare star that will be returned before it gets to this part of the code.
Fermi has this most_likely_index where it tells you what it thinks it is but it doesn't feel well documented to me so for now I'll keep it in trigger logic where I can keep a log (trigger_message). It may be useful to use the most_likely_index to define other source types in the future.